README.md

```diff
@@ -7,7 +7,7 @@ Ballerina connector for Microsoft Outlook [Mail](https://docs.microsoft.com/en-u
 access Microsoft Cloud service resources. This version of the connector supports most of the important mail service related operations.
  
 For more information, go to the module(s).
-- [ballerinax/microsoft.outlook.mail](https://github.com/SanduDS/module-ballerinax-microsoft.outlook.mail/tree/main/outlookmail)
+- [microsoft.outlook.mail](outlookmail/Module.md)
```

Comment:
Let's rephrase the above paragraph as "Ballerina connector for Microsoft Outlook [Mail](https://docs.microsoft.com/en-us/graph/api/resources/mail-api-overview?view=graph-rest-1.0) provides you the capability to access Microsoft Cloud service resources. The connector internally uses the Microsoft Graph that provides REST web APIs."

Reply:
Let's rephrase the above sentence as : "This version of the connector supports most of the mail service-related operations."